fedimint-load-test-tool/src/common.rs

```diff
@@ -302,92 +298,50 @@ pub async fn remint_denomination(
     Ok(())
 }
 
-pub struct FedimintCli;
-impl ToCmdExt for FedimintCli {
-    type Fut = std::future::Ready<devimint::util::Command>;
+fn get_command_for_alias(alias: &str, default: &str) -> devimint::util::Command {
```

Comment:
That's no really an "alias", but an env var?

Reply:
Yeah. I uses alias because for each of theses variables there is a corresponding alias, like: `alias fedimint-cli='$FM_MINT_CLIENT'`